integreat_cms/core/settings.py

```diff
@@ -888,6 +883,21 @@
 )
 
 
+#################
+# INTERNAL URLS #
+#################
+
+#: The URLs which are treated as internal in TinyMCE custom link plugin
+INTERNAL_URLS = (
+    ALLOWED_HOSTS
+    + [WEBAPP_URL, WEBSITE_URL]
+    + [
+        x.strip(x)
```

Comment:
```suggestion
        x.strip()
```
Oh, `x.strip(x)` will always return an empty string.

Reply:
Whoops! Good catch, also didn't see that.